.github/workflows/pull-request-check-rust-api.yaml

```diff
@@ -101,8 +101,15 @@ jobs:
       # We won't be running any of the regular regression tests, as these are covered
       # by the other jobs already present in `pull-request-checks.yaml`.
       - name: Run Rust API tests
+        env:
+          # This is needed because an issue on the CI runner is preventing the link-cplusplus crate
+          # cxx.rs depends on to select the correct default (libc++, the clang++ standard library)
+          # on this platform.
+          # (https://docs.rs/cc/latest/cc/struct.Build.html#method.cpp_link_stdlib)
+          CXXSTDLIB: c++
         run: |
-          VERSION=$(cat src/config.inc | python3 -c "import sys,re;line = [line for line in sys.stdin if re.search('CBMC_VERSION = (\d+\.\d+\.\d+)', line)];sys.stdout.write(re.search('CBMC_VERSION = (\d+\.\d+\.\d+)', line[0]).group(1))")
+          export MACOSX_DEPLOYMENT_TARGET=10.15
+          VERSION=$(cat src/config.inc | python3 -c "import sys,re;line = [line for line in sys.stdin if re.search(r'CBMC_VERSION = (\d+\.\d+\.\d+)', line)];sys.stdout.write(re.search(r'CBMC_VERSION = (\d+\.\d+\.\d+)', line[0]).group(1))")
           cd src/libcprover-rust;\
           cargo clean;\
-          CBMC_INCLUDE_DIR=../../${{env.default_include_dir}} CBMC_LIB_DIR=../../${{env.default_build_dir}}/lib CBMC_VERSION=$VERSION cargo test -- --test-threads=1
+          CBMC_INCLUDE_DIR=../../${{env.default_include_dir}} CBMC_LIB_DIR=../../${{env.default_build_dir}}/lib CBMC_VERSION=$VERSION cargo test --verbose -- --test-threads=1
```

Comment:
Probably no longer necessary?

Reply:
You're right, this was just for debugging. Will go now.